lib/src/verison_constants.dart

```diff
@@ -0,0 +1,17 @@
+class VersionConstants {
+  VersionConstants._();
+
+  static const String flutterSdkVersion = '0.3.2';
+
+  static const String iosSdkVersion = '0.3.1';
```

Comment:
Can we take these parameters from the iOS SDK and Android SDK directly instead of hardcoding?

Reply:
Got it, looks fine. Please make sure that we add this step to our release process that this file needs to be updated with each release.

Reply:
yes sure